client/a8c-for-agencies/components/items-dashboard/item-preview-pane/item-preview-pane-header/index.tsx

```diff
@@ -116,45 +119,39 @@ export default function ItemPreviewPaneHeader( {
 									''
 								) }
 							</div>
-							{ config.isEnabled( 'hosting-overview-refinements' ) &&
-								( wpVersion || phpVersion ) && (
-									<div className="item-preview__header-env-data">
-										{ wpVersion && (
-											<div className="item-preview__header-env-data-item">
-												WordPress{ ' ' }
-												{ isAtomic ? (
-													<Button
-														className="item-preview__header-env-data-item-link"
-														href={ `/hosting-config/${ selectedSite?.domain }#wp` }
-														onClick={ handleWpVersionClick }
-													>
-														{ wpVersion }
-														{ wpVersionName && isStagingSite && (
-															<span className="item-preview__header-env-data-item-link-description">
-																({ wpVersionName })
-															</span>
-														) }
-													</Button>
-												) : (
-													wpVersion
-												) }
-											</div>
-										) }
-										{ phpVersion && (
-											<div className="item-preview__header-env-data-item">
-												PHP{ ' ' }
-												<Button
-													className="item-preview__header-env-data-item-link"
-													onClick={ handlePhpVersionClick }
-													href={ `/hosting-config/${ selectedSite?.domain }#php` }
-												>
-													{ phpVersion }
-												</Button>
-											</div>
-										) }
-									</div>
-								) }
+
+							{ shouldDisplayVersionNumbers && (
+								<div className="item-preview__header-env-data">
+									{ wpVersion && (
+										<div className="item-preview__header-env-data-item">
+											WordPress{ ' ' }
+											<a
+												className="item-preview__header-env-data-item-link"
+												href={ `/hosting-config/${ selectedSite?.domain }#wp` }
+												onClick={ handleWpVersionClick }
+											>
+												{ wpVersion }
+												{ wpVersionName && isStagingSite && <span> ({ wpVersionName })</span> }
+											</a>
```

Comment:
@nightnei, I made an additional change here, namely to use a plan `<a>` instead of a `<Button>`. This means we don't need to explicitly style the `<span>`, we can just use regular white space in the HTML to separate the version number and the "version track".

The f38a7ba1689ef292947f3232d0544a45450f3492 diff should clarify the change even more.